hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/token/ContainerTokenIdentifier.java

```diff
@@ -87,6 +87,16 @@ public void readFields(DataInput in) throws IOException {
     this.containerID = ContainerID.getFromProtobuf(proto.getContainerId());
   }
 
+  @Override
+  public void readFromByteArray(byte[] bytes) throws IOException {
+    ContainerTokenSecretProto proto =
+        ContainerTokenSecretProto.parseFrom(bytes);
+    setSecretKeyId(ProtobufUtils.fromProtobuf(proto.getSecretKeyId()));
+    setExpiry(Instant.ofEpochMilli(proto.getExpiryDate()));
+    setOwnerId(proto.getOwnerId());
+    this.containerID = ContainerID.getFromProtobuf(proto.getContainerId());
```

Comment:
Please extract this part to a method and reuse in `readFields`:

https://github.com/apache/ozone/blob/41cb6d0ea95aeddc59e98005529204278e9be02f/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/token/ContainerTokenIdentifier.java#L84-L87